libavfilter/dnn/dnn_backend_openvino.c

```diff
@@ -61,6 +65,25 @@ static DNNDataType precision_to_datatype(precision_e precision)
     }
 }
 
+static DNNReturnType parse_options(OVContext *dst, const char *src_opt, ie_available_devices_t *a_dev)
+{
+    if (av_opt_set_from_string(dst, src_opt, NULL, "=", "&") < 0) {
```

Comment:
if av_opt_set_from_string is enough, just call it within ff_dnn_load_model_ov, and remove function parse_options

Reply:
OK, got you